src/main/java/mate/academy/spring/validation/annotation/Password.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.validation.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import mate.academy.spring.validation.PasswordValidator;
+
+@Constraint(validatedBy = PasswordValidator.class)
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Password {
+    String message() default "Invalid password";
```

Comment:
you could create more informative message here. ex:
"Password should contain at least on upper and lower case letter, one digit......."  something like this